src/test/java/wooteco/subway/web/member/MemberAPITest.java

```diff
@@ -0,0 +1,43 @@
+package wooteco.subway.web.member;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.restdocs.RestDocumentationContextProvider;
+import org.springframework.restdocs.RestDocumentationExtension;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import org.springframework.web.context.WebApplicationContext;
+import org.springframework.web.filter.ShallowEtagHeaderFilter;
+import wooteco.subway.domain.member.MemberRepository;
+import wooteco.subway.infra.JwtTokenProvider;
+import wooteco.subway.service.member.MemberService;
+
+import static org.springframework.restdocs.mockmvc.MockMvcRestDocumentation.documentationConfiguration;
+
+@ExtendWith(RestDocumentationExtension.class)
+@SpringBootTest
+@AutoConfigureMockMvc
+public class MemberAPITest {
```

Comment:
```suggestion
public class MemberApiTest {
```

네이밍에 정답은 없지만 저는 카멜케이스로 해주는 편인데요.
HttpApiUrl과 같은 이름이라면 어떻게 만들 수 있을까요?

Reply:
카멜케이스가 더 좋을 것 같습니다. 모든 줄임말을 대문자로 표기하는것은 가독성을 떨어뜨릴 수 있을 것 같네요. 수정하였습니다.